Sources/Utils/Log.cpp

```diff
@@ -147,8 +148,10 @@ void vLog(int level, char const *classname, char const *funcname,
   fputs(ss.str().c_str(), sOutputStream);
   fflush(sOutputStream);
 
-  if (level == kLogLevelFatal)
+  if (level == kLogLevelFatal) {
+    ds2::Utils::PrintBacktrace();
     abort();
+  }
```

Comment:
Correct :D

On Wed, Apr 6, 2016 at 10:33 AM, Francis Ricci <notifications@github.com>
wrote:

> In Sources/Utils/Log.cpp
> <https://github.com/facebook/ds2/pull/253#discussion_r58747904>:
>
> >      abort();
> > +  }
>
> And this git blame is why @a20012251 <https://github.com/a20012251> likes
> curly braces on everything. I'm starting to understand [image: :laughing:]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/253/files/1e296ca1bf4f64840f2696500f089169b6f2fadd..5060750f1af5f073971d91bda1242321dc330c21#r58747904>
>



-- 
Stephane Sezer